students/Harry_Maher/Session04/_kata_fourteen.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+"""
+Kata Fourteen: Tom Swift Under Milk Wood
+
+For this kata, try implementing a trigram algorithm that generates a couple of 
+hundred words of text using a book-sized file as input. Project Gutenberg is 
+a good source of online books
+
+
+I found a book entitled: THE “SURE to RISE” COOKERY BOOK. An old pun 
+from a 1914! Hopefully it spits out a good bread recipe for my bread club...
+
+-----
+Here are some good recipes it came up with:
+
+DATE CAKE. ½ lb. dates 1½ breakfastcups of flour piled up equal 1 lb. of steak ½ lb. 
+flour 1 teaspoonful flour 1 pint milk
+
+BATH BUNS. ½ cupful sweet milk 1 teaspoon lemon juice if desired, place in glass 
+dish, and cover with vinegar, boil slowly 4 or 5 hours.
+
+PLAIN PUFF PASTRY. ½ lb. butter, then mix egg and milk together, and add the eggs, 
+then flour and baking powder lightly and quickly, pour into greased tin, and bake 
+in quick oven. (When fully risen open door to let out the paste to a cream, then 
+beat in eggs. Mix currants with flour and baking powder, mix well, then add the milk
+
+ENGLISH QUEEN CAKES. 4 eggs Grate only the yellow part of the custard—if you use 
+Edmonds’ you can take the rest for granted. Let the children have all the dry 
+ingredients, add eggs well beaten, and enough milk to desired paste, roll out, 
+cover with a fork, and add to mixture. Grease two small dishes, fill half full 
+of fresh fruit, pour over the fire with the milk slightly warmed, and bake in 
+hot oven for about half-an-hour.
+------
+
+input: book.txt 
+output: my_new_book.txt (300 words--not a real book. Really a couple of recipes, hopefully.)
+
+
+"""
+import sys
+import random
+
+def main(book_input):
+    """ Taking a book as input, implement a trigram algorithm that generates 300 words of text using a book-sized file as input"""
+    trigram_d = trigram_dictionary_builder(book_input)
```

Comment:
Since you're calling another function here that you haven't defined yet, it might be better to put this function after trigram_dictionary_builder()

Reply:
Just for style purposes....